Makefile

```diff
@@ -295,3 +295,6 @@ docs-serve-dev:
 	  -v "$(CURDIR):/k0s:ro" \
 	  -p '$(DOCS_DEV_PORT):8000' \
 	  k0sdocs.docker-image.serve-dev
+
+# Helper target to printe internal make vars
+print-%  : ; @echo $($*)
```

Comment:
We have the [`vars.sh`](https://github.com/k0sproject/k0s/blob/v1.25.2%2Bk0s.0/vars.sh) script for that nowadays.

```patch
diff --git a/.github/workflows/check-network.yaml b/.github/workflows/check-network.yaml
index aebe6a3a4..d25f191a4 100644
--- a/.github/workflows/check-network.yaml
+++ b/.github/workflows/check-network.yaml
@@ -96,7 +96,7 @@ jobs:
         id: k0s_build
         run: |
           make build
-          KUBERNETES_VERSION=$(make print-kubernetes_version)
+          KUBERNETES_VERSION="$(./vars.sh kubernetes_version)"
           echo KUBERNETES_VERSION="$KUBERNETES_VERSION" >> $GITHUB_OUTPUT
         working-directory: ./
 
```